movingpandas/geometry_utils.py

```diff
@@ -34,6 +35,19 @@ def measure_distance_euclidean(point1, point2):
         raise TypeError("Only Points are supported as arguments, got {} and {}".format(point1, point2))
     return point1.distance(point2)
 
+def measure_distance_geodesic(point1, point2):
+    """
+    Return the geodesic distance between two points as a float of (SI) unit meters.
```

Comment:
This function assumes, lat/lon points, right?

Reply:
Yes, the function inputs (Shapely.geometry) Point objects. The x and y are assumed to be latitude / longitude.
In addition, it should be noted that the method ```distance.distance((lat1,lon1), (lat2,lon2))``` defaults to the WGS-84 datum. WGS-84 is used by GPS and is likely the most common datum.

The datum can be changed like so ```distance.geodesic(point1, point2, ellipsoid='GRS-67').meters)```